index.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Product Landing Page</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header id="header">
+        <nav id="nav-bar">
+        <ul>
+            <li><img id="header-img" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png"/></li>
```

Comment:
It is a better practice to use `<div> `tag instead of `<img>` and style image as background image.
here is an example,
in the HTML file;
`<div class="header-img"</div>`

In the CSS;
```
.header-img{
background-image=url("./Image/header")
} 
```